mlflow/tracking/client.py

```diff
@@ -1671,10 +1671,10 @@ def log_table(
                 "toxicity": [0.0, 0.0],
             }
             client = MlflowClient()
```

Comment:
I think you need to create the client after `start_run()`, does the current code work?

Reply:
Yep, I think it works: https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#notebook/3429265658202847 but I updated it so that it is inside the start_run() now for better practice